lib/improver/nowcasting/pysteps_interfacing.py

```diff
@@ -0,0 +1,145 @@
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# (C) British Crown Copyright 2017-2019 Met Office.
+# All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions are met:
+#
+# * Redistributions of source code must retain the above copyright notice, this
+#   list of conditions and the following disclaimer.
+#
+# * Redistributions in binary form must reproduce the above copyright notice,
+#   this list of conditions and the following disclaimer in the documentation
+#   and/or other materials provided with the distribution.
+#
+# * Neither the name of the copyright holder nor the names of its
+#   contributors may be used to endorse or promote products derived from
+#   this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+# ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+# LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+# CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+# SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+# INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+# CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+# ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+# POSSIBILITY OF SUCH DAMAGE.
+"""
+Functions to assist with interface to pysteps library
+"""
+
+import numpy as np
+
+from improver.utilities.load import load_cube
+from improver.utilities.spatial import check_if_grid_is_equal_area
+
+
+class PystepsImporter(object):
+    """
+    Class containing methods to import from IMPROVER NetCDF file or cube
+    and return data and metadata.  This is to allow use of the pysteps
+    plotting functions in investigations only, and should not be required for
+    any of the pysteps processing routines.
```

Comment:
> This is to allow use of the pysteps plotting functions in investigations only, and should not be required for any of the pysteps processing routines.

Does this code belong in the improver lib?

Reply:
Probably not in the long term.  However since this is a feature branch partly for case studies, and given Ben demonstrated the usefulness of pysteps plotting routines in his initial notebook, I thought it would be useful to have available for those investigations.  I don't anticipate merging this into master when we've decided on a nowcasting method.

Is there a standard way to flag code for removal, or that should not be merged into master?

Reply:
OK, that's good. There's no way to prevent it being merged, but marking the branch name, eg: feature/pysteps_interface can be used, to indicate as much.

Reply:
I've added something to the module docstring.